### ISSUE TITLE
This app doesn't launch on Windows.

### ISSUE THREAD

**Issue by OfficialIncubo** [tedsteen/Spotiamp#34]
Hey! I have installed your Spotiamp using the .exe installer. I connected to Spotify, then I launched Spotiamp, then nothing worked. Same happened for the first time (it only appeared the Spotify connect page).
How can I get it work?

**Comment by tedsteen** [comment]
You need a Spotify premium account or else it won't work. If you do, then try the latest version I just pushed. If you still have problems then we need to dig a bit deeper :)

**Comment by tedsteen** [comment]
Will close this one soon unless there is still a problem

**Comment by OfficialIncubo** [comment]
I don't have Spotify Premium, but I tried and still doesn't run.
I need to work with both Spotify Free and Premium.

**Comment by tedsteen** [comment]
> I don't have Spotify Premium, but I tried and still doesn't run.
> I need to work with both Spotify Free and Premium.

Yeah at this point in time as far as I know spotify doesn't let you use the api with a free account (not through the librespot library that Spotiamp use at least) perhaps this will change (or has changed already) Something that need to be looked into.
